hist/histv7/test/fill.cxx

```diff
@@ -179,9 +179,9 @@ TEST(HistFillTest, FillNCoordsEntries)
 {
    ROOT::Experimental::RH1F hist({100, 0., 1});
    EXPECT_EQ(0, hist.GetEntries());
-   hist.FillN({0.1111, 0.2222, 0.3333});
+   hist.FillN({{0.1111, 0.2222, 0.3333}});
    EXPECT_EQ(3, hist.GetEntries());
-   hist.FillN({0.1111, 0.3333});
+   hist.FillN({{0.1111, 0.3333}});
```

Comment:
Could you remove this file from this PR? This will make it fail on clang - I will provide a solution that works for everyone, in a separate PR.